javalanglib/src/main/scala/java/lang/Iterable.scala

```diff
@@ -13,7 +13,14 @@
 package java.lang
 
 import java.util.Iterator
+import java.util.function.Consumer
 
 trait Iterable[T] {
   def iterator(): Iterator[T]
+
+  def forEach(action: Consumer[_ >: T]): Unit = {
```

Comment:
You'll need to add the `@JavaDefaultMethod` annotation for tests to pass on 2.10 and 2.11.